src/Pivot/Modify.elm

```diff
@@ -67,8 +84,13 @@ appendGoR val =
     appendGoL val |> mirror
 
 
-appendList : List a -> Pivot a -> Pivot a
-appendList xs =
+appendListL : List a -> Pivot a -> Pivot a
+appendListL xs =
+    mapL_ ((++) xs)
```

Comment:
The auto-upgrade tool replaces `(,) x` with `\a -> (x,a)`, but leaves `(++) x` unchanged?